hypothesis-python/tests/cover/test_reflection.py

```diff
@@ -622,3 +624,33 @@ def f(source):
 
     with pytest.raises(ValueError, match="SyntaxError because `from` is a keyword"):
         get_signature(f)
+
+
+def test_param_is_called_within_func():
+    def f(any_name):
+        any_name()
+
+    assert is_func_param_called_within(f, "any_name") is True
```

Comment:
Asserting on `is` is generally a little verbose for us; prefer `assert is_called(...)` or `assert not is_called(...)`.